src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ConfigurationTests.cs

```diff
@@ -0,0 +1,268 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tools.Monitor;
+using Microsoft.Diagnostics.Tools.Monitor.Commands;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Diagnostics.Monitoring.Tool.UnitTests
+{
+    public class ConfigurationTests
+    {
+        private readonly ITestOutputHelper _outputHelper;
+
+        // This needs to be updated and kept in order for any future configuration sections
+        private readonly List<string> orderedConfigurationKeys = new()
```

Comment:
You might want to pull this list out of `RootOptions` with reflection and avoid needing to keep this up to date.

Reply:
I like the sentiment, although it's not the same list as what's in `RootOptions` (and the ordering, at least in the current design, does need to be the same). That being said, if there's a way to avoid needing to keep this up to date that would reduce upkeep